common/js/rechat.js

```diff
@@ -367,16 +372,21 @@ ReChat.Playback.prototype._replaceEmoticonsByRanges = function(text, emotes) {
 ReChat.Playback.prototype._formatChatMessage = function(messageData) {
   var userColor = this._colorForNickname(messageData.from, messageData.usercolor);
   var line = $('<div>').addClass('chat-line rechat-chat-line rechat-user-' + messageData.from);
+  var badges = $('<span>').addClass('float-left').addClass('badges');
+
   // Add data attributes
   line.attr('data-sender', messageData.from);
   line.attr('data-room', messageData.to.substring(1));
+  // Apply possible sub/turbo/usertype/broadcaster badges to message
+  this._applyMessageBadges(messageData, badges);
   // From line
   var from = $('<span>').addClass('from').css({
         'color': userColor,
         'font-weight': 'bold'
       }),
       colon = $('<span>').addClass('colon'),
-      message = $('<span>').addClass('message').css({ 'word-wrap': 'break-word' }),
+      message = $('<span>').addClass('message').css({ 'word-wrap': 'break-word' })
+                           .attr('style', 'line-height: 20px!important;'); // BTTV style overriding
```

Comment:
You can't set the `!important` property using `$.css()`. You have to resort to `element.style` or `$.attr()`. Also, the computed values for `.message` include `word-wrap: break-word;`.